README.md

```diff
@@ -13,33 +13,29 @@ A module for the [MagicMirror²](https://magicmirror.builders) to display air qu
 
 ## Installation
 
-1. Navigate to your MagicMirror `modules` folder:
+Just clone the module into your modules directory:
 
-    ```bash
-    cd ~/MagicMirror/modules
-    ```
-
-2. Clone the repository:
+```bash
+cd ~/MagicMirror/modules
+git clone https://github.com/PierreGode/MMM-airquality
+```
 
-    ```bash
-    git clone https://github.com/PierreGode/MMM-airquality
-    ```
+## Update
 
-3. Navigate to the module folder:
+Just enter the module's directory and pull the update:
 
-    ```bash
-    cd MMM-airquality
-    ```
+```bash
+cd ~/MagicMirror/modules/MMM-NHL
```

Comment:
Sorry for the NHL :facepalm: But you've already fixed it :+1:

Reply:
Haha no worries. Thanks for the updates 😊